helm_deploy/values-dev.yaml

```diff
@@ -11,18 +11,16 @@ image:
 
 ingress:
   enabled: true
-  enable_whitelist: true
```

Comment:
Could we also remove the `if .Values.ingress.enable_whitelist` block in the `ingress.yaml`, please?

Reply:
yep